src/app/components/AccountMenu/index.tsx

```diff
@@ -85,12 +85,12 @@ function AccountMenu({ showOptions = true }: Props) {
             <div
               className={`flex-auto mx-2 py-3 overflow-hidden max-w-[14rem] text-left`}
             >
-              <p
+              <div
                 title={title || ""}
                 className="text-sm font-medium text-gray-700 dark:text-neutral-400 text-ellipsis overflow-hidden whitespace-nowrap"
               >
-                {title || <Skeleton />}
-              </p>
+                {loading ? <SkeletonLoader className="w-20" /> : title || "⚠️"}
```

Comment:
Currently errors while switching accounts result in an empty account name. I've added a warning icon for now: 

![image](https://user-images.githubusercontent.com/100827540/234711080-feeaef77-abbb-45ba-a207-8e9cb1e6f899.png)

Also I created a followup issue to tackle this problem: #2377